tests/integration/cli_mint_proposal_test.go

```diff
@@ -27,36 +28,52 @@ var _ = Describe("Integration - Mint coins to given address", func() {
 		By("sending a SubmitProposal transaction from `validator0` container")
 		res, err := cli.SubmitProposalTx(cli.Operator0, "proposal.json", cli.CliGasParams)
 		Expect(err).To(BeNil())
-		Expect(res.Code).To(BeEquivalentTo(0))
-	})
+		res, err = cli.QueryTxn(res.TxHash)
+		Expect(err).To(BeNil())
 
-	It("should wait for the proposal submission to be included in a block", func() {
-		By("getting the current block height")
-		currentHeight, err := cli.GetCurrentBlockHeight(cli.Validator0, cli.CliBinaryName)
+		proposal_id, err := cli.GetProposalID(res.RawLog)
+
+		fmt.Println("The proposal id>>>>>>>>>>>>>>>>>>>>>>>>>>>.", proposal_id)
+		Proposal_id = proposal_id
 		Expect(err).To(BeNil())
 
-		By("waiting for the proposal to be included in a block")
-		err = cli.WaitForChainHeight(cli.Validator0, cli.CliBinaryName, currentHeight+1, 2)
+		Expect(res.Code).To(BeEquivalentTo(0))
+	})
+	It("keys list", func() {
+		By("keys list in validator 0")
+		keys, err := cli.QueryKeysList()
 		Expect(err).To(BeNil())
+		fmt.Println("keys arE>>>>>>>>>>>>>>>>>", keys)
 	})
 
+	// It("should wait for the proposal submission to be included in a block", func() {
+	// 	By("getting the current block height")
+	// 	currentHeight, err := cli.GetCurrentBlockHeight(cli.Validator0, cli.CliBinaryName)
+	// 	Expect(err).To(BeNil())
+
+	// 	By("waiting for the proposal to be included in a block")
+	// 	err = cli.WaitForChainHeight(cli.Validator0, cli.CliBinaryName, currentHeight+1, 2)
+	// 	Expect(err).To(BeNil())
+	// })
+
```

Comment:
don't we need this?

Reply:
Actually this was causing the proposal inactive.So i have commented this code block